Misc/NEWS.d/next/Library/2024-12-12-07-27-51.gh-issue-127847.ksfNKM.rst

```diff
@@ -0,0 +1 @@
+Fix the position when doing interleaved seeks and reads in uncompressed unencrypted zip files
```

Comment:
It's good to have some context for what was fixed (also some typos; english is hard).
```suggestion
Fix the position when doing interleaved seeks and reads in uncompressed, unencrypted zip files returned by :meth:`zipfile.ZipFile.open`.
```